packages/core-snapshots-cli/lib/db/queries/transactions/timestamp-range.sql

```diff
@@ -0,0 +1,19 @@
+SELECT
+  ID,
```

Comment:
Properties in queries should be all lower-case, only SQL keywords should be uppercase.